docs/ref/contrib/admin/index.txt

```diff
@@ -35,7 +35,7 @@ If you're not using the default project template, here are the requirements:
 #. Configure a :class:`~django.template.backends.django.DjangoTemplates`
    backend in your :setting:`TEMPLATES` setting with
    ``django.contrib.auth.context_processors.auth``,
-   ``django.contrib.messages.context_processors.request``, and
+   ``django.template.context_processors.request``, and
```

Comment:
Please fix it also in the `versionchanged` annotation, line 45.

Reply:
Just below, and I forgot ^^